### ISSUE TITLE
Position on screen is semi-off screen when using two displays with each a different resolution.

### ISSUE THREAD

**Issue by benjaminnaesen** [appetizermonster/hain#267]
Here is an example: http://prntscr.com/jow9y4

As visible, the position of the Hain launcher hangs in between the 2 screens. 

This happens with the usage of two displays:
Resolutions: 1080p and 4K

**Comment by dannya** [comment]
@benjaminnaesen is this on your 4k monitor? I'm surprised the window is so big, the default size is only ~800 wide / ~550 high.

**Comment by benjaminnaesen** [comment]
Yes this in on my 4K monitor. (This doesn't happen when not using 2 screens.) As you can see in this picture (http://prntscr.com/jowmpo) It kinda shows at the bottom of the non 4K external monitor too. :D

**Comment by dannya** [comment]
@benjaminnaesen are you able to run the version from Git, branch develop? 

I've just merged some changes that might impact this that I'd like your feedback on. Thanks.

**Comment by benjaminnaesen** [comment]
Let me try.
		
		

		Get Outlook for iOS
	




On Mon, Jun 11, 2018 at 10:43 PM +0200, "Danny Allen" <notifications@github.com> wrote:












@benjaminnaesen are you able to run the version from Git, branch develop?


I've just merged some changes that might impact this that I'd like your feedback on. Thanks.

**Comment by benjaminnaesen** [comment]
I'll have to try at work tomorrow, don't have a second monitor here. :)

**Comment by dannya** [comment]
@benjaminnaesen have you been able to try out the latest code from develop?

**Comment by thedonquixotic** [comment]
@dannya I seem to be running into the same problem. It wasn't doing this before but now it is. I'm on a surface book using a MS dock to extend 2 old TVS as monitors through HDMI. I'm using whatever the most recent version that can be installed from executable. .66 I think.

**Comment by thedonquixotic** [comment]
Oh yeah also I'm using Display Fusion Pro to do my multi monitor set up which might be part of the issue. I'll test that. 

EDIT: Checked it. Turned off DF and it was still wonky. But restarted Hain and it was fine. I think it's just that I have to restart it after I plug in my monitors. It's good now. Maybe this was the same thing for Ben. 

Thank you for creating this awesome app and continuing to develop it!

**Comment by thedonquixotic** [comment]
actually..... nvm. It's still acting wonky.

**Comment by Kyle-Helmick** [comment]
I hate to revive dead issues buuuuuuuuut, yea I'm having the same issue. I'm finding that alternating which monitor hain is opened on makes it grow in size until it's stretching across both screens.

I don't know a lot about electron apps but could we make the window dimensions constants instead of a datatype that can be changed?
